apps/src/gamelab/GameLabSprite.js

```diff
@@ -106,38 +106,24 @@ module.exports.createSprite = function (x, y, width, height) {
   });
 
   // p5.play stores width unscaled, but users in
-  // Game Lab should access and set a scaled version.
-  Object.defineProperty(s, 'width', {
-    enumerable: true,
-    configurable: true,
-    get: function () {
-      if (s._internalWidth === undefined) {
-        return 100;
-      } else {
-        return s._internalWidth * this.scale;
-      }
-    },
-    set: function (value) {
-      s._internalWidth = value / this.scale;
+  // Game Lab should have access to a scaled version.
+  s.getScaledWidth = function () {
+    if (s.width === undefined) {
+      return 100 * this.scale;
```

Comment:
Yes, you're right. This was leftover from overriding height and width. I'll remove.